### PR TITLE
feat(forms-web-app): rename appellant-submission to appeal-householde…

### PR DESCRIPTION
**Pull request by davehlewis**
## Ticket Number

AS-1516

## Description of change
Rename appellant-submission to appeal-householder-decision. Have left the folder structures intact where possible. Due to the way things were setup, needed to rename views folder as this was used for routing in the task service.

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [X] My commit history in this PR is linear
- [X] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/forms-web-app/src/routes/index.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const guidancePagesRouter = require('./guidance-pages');
 router.use('/', homeRouter);
 router.use(guidancePagesRouter);
 router.use('/cookies', cookieRouter);
-router.use('/appellant-submission', appellantSubmissionRouter);
+router.use('/appeal-householder-decision', appellantSubmissionRouter);
```
Comment (a6software) [inline packages/forms-web-app/src/routes/index.js:14]: could the router be renamed here - `appealHouseholderDecisionRouter` ?

File: e2e-tests/cypress/integration/common/appeal-submission-navigation.js [hunk 2]
```diff
@@ -1,41 +1,41 @@
 import { Then } from 'cypress-cucumber-preprocessor/steps';
 
 Then('the user is navigated to "Appeal tasks"', () => {
-  cy.userIsNavigatedToPage('/appellant-submission/task-list');
+  cy.userIsNavigatedToPage('/appeal-householder-decision/task-list');
```
Comment (a6software) [inline e2e-tests/cypress/integration/common/appeal-submission-navigation.js:4]: please can this file be renamed to `appeal-householder-decision-navigation` ?

File: e2e-tests/cypress/integration/appeal-submission-tasklist/appeal-submission-tasklist.js [hunk 3]
```diff
@@ -6,47 +6,47 @@ function getTask(sectionTaskName) {
   switch (sectionTaskName) {
     case 'About you - Your details':
       name = 'yourDetails';
-      url = '/appellant-submission/who-are-you';
+      url = '/appeal-householder-decision/who-are-you';
```
Comment (a6software) [inline e2e-tests/cypress/integration/appeal-submission-tasklist/appeal-submission-tasklist.js:9]: Please can this file & dir be renamed to 

`appeal-householder-decision-tasklist/appeal-householder-decision-tasklist.js`

File: e2e-tests/cypress/integration/appeal-submission-save-and-continue-navigation-appeal-site/appeal-submission-save-and-continue-navigation-appeal-site.js [hunk 4]
```diff
@@ -18,7 +18,7 @@ Given('the "Site ownership" is not wholly owned', () => {
   cy.answerDoesNotOwnTheWholeAppeal();
   cy.clickSaveAndContinue();
 
-  cy.userIsNavigatedToPage('/appellant-submission/site-ownership-certb');
+  cy.userIsNavigatedToPage('/appeal-householder-decision/site-ownership-certb');
```
Comment (a6software) [inline e2e-tests/cypress/integration/appeal-submission-save-and-continue-navigation-appeal-site/appeal-submission-save-and-continue-navigation-appeal-site.js:21]: Please can this file and dir be renamed to reflect `appeal-householder-decision` ? thanks

File: e2e-tests/cypress/support/appeal-navigation-confirmation/appellant-submission/confirmNavigationApplicantNamePage.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
 module.exports = () => {
-  cy.url().should('include', '/appellant-submission/applicant-name');
+  cy.url().should('include', '/appeal-householder-decision/applicant-name');
```
Comment (a6software) [inline e2e-tests/cypress/support/appeal-navigation-confirmation/appellant-submission/confirmNavigationApplicantNamePage.js:2]: please can this dir be renamed from `appellant-submission` to `appeal-householder-decision` ? thanks

**Review by Fenisay — APPROVED** [review]
LGTM

File: e2e-tests/cypress/support/appeal-submission-access-to-appeal-site/confirmAccessSiteWasAccepted.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 module.exports = () => {
   // confirm we are in the right place
-  cy.url().should('include', '/appellant-submission/task-list');
+  cy.url().should('include', '/appeal-householder-decision/task-list');
```
Comment (a6software) [inline e2e-tests/cypress/support/appeal-submission-access-to-appeal-site/confirmAccessSiteWasAccepted.js:3]: please can the dir here be updated to `appeal-householder-decision-access-to-appeal-site` ? thanks

File: e2e-tests/cypress/support/appeal-submission-appeal-site-address/confirmSiteAddressWasAccepted.js [hunk 7]
```diff
@@ -1,6 +1,6 @@
 module.exports = () => {
   // confirm we are in the right place
-  cy.url().should('include', '/appellant-submission/site-ownership');
+  cy.url().should('include', '/appeal-householder-decision/site-ownership');
```
Comment (a6software) [inline e2e-tests/cypress/support/appeal-submission-appeal-site-address/confirmSiteAddressWasAccepted.js:3]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: e2e-tests/cypress/support/appeal-submission-appeal-site-ownership/confirmOtherOwnersAsked.js [hunk 8]
```diff
@@ -1,6 +1,6 @@
 module.exports = () => {
   // confirm we are in the right place
-  cy.url().should('include', '/appellant-submission/site-ownership-certb');
+  cy.url().should('include', '/appeal-householder-decision/site-ownership-certb');
```
Comment (a6software) [inline e2e-tests/cypress/support/appeal-submission-appeal-site-ownership/confirmOtherOwnersAsked.js:3]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: e2e-tests/cypress/support/appeal-submission-site-health-and-safety-issues/confirmHealthAndSafetyPage.js [hunk 9]
```diff
@@ -1,4 +1,4 @@
 module.exports = () => {
-  cy.url().should('contain', '/appellant-submission/site-access-safety');
+  cy.url().should('contain', '/appeal-householder-decision/site-access-safety');
```
Comment (a6software) [inline e2e-tests/cypress/support/appeal-submission-site-health-and-safety-issues/confirmHealthAndSafetyPage.js:2]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: e2e-tests/cypress/support/appeal-submission-tasklist/confirmUserPresentedWithProvidePlanningApplicationNumber.js [hunk 10]
```diff
@@ -1,4 +1,4 @@
 module.exports = () => {
-  cy.url().should('contain','/appellant-submission/application-number');
+  cy.url().should('contain','/appeal-householder-decision/application-number');
```
Comment (a6software) [inline e2e-tests/cypress/support/appeal-submission-tasklist/confirmUserPresentedWithProvidePlanningApplicationNumber.js:2]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: e2e-tests/cypress/support/appellant-submission-check-your-answers/confirmCheckYourAnswersPage.js [hunk 11]
```diff
@@ -1,4 +1,4 @@
 module.exports = () => {
-  cy.url().should('include', '/appellant-submission/check-answers');
+  cy.url().should('include', '/appeal-householder-decision/check-answers');
```
Comment (a6software) [inline e2e-tests/cypress/support/appellant-submission-check-your-answers/confirmCheckYourAnswersPage.js:2]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: e2e-tests/cypress/support/appellant-submission-decision-letter/confirmDecisionLetterRejectedBecause.js [hunk 12]
```diff
@@ -1,6 +1,6 @@
 module.exports = (errorMessage) => {
   // confirm we are in the right place
-  cy.url().should('include', '/appellant-submission/upload-decision');
+  cy.url().should('include', '/appeal-householder-decision/upload-decision');
```
Comment (a6software) [inline e2e-tests/cypress/support/appellant-submission-decision-letter/confirmDecisionLetterRejectedBecause.js:3]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: e2e-tests/cypress/support/appellant-submission-planning-application-number/confirmPlanningApplicationNumberHasNotUpdated.js [hunk 13]
```diff
@@ -1,5 +1,5 @@
 module.exports = () => {
-  cy.visit('/appellant-submission/application-number');
+  cy.visit('/appeal-householder-decision/application-number');
```
Comment (a6software) [inline e2e-tests/cypress/support/appellant-submission-planning-application-number/confirmPlanningApplicationNumberHasNotUpdated.js:2]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: e2e-tests/cypress/support/appellant-submission-supporting-documents/confirmSupportingDocumentRejectedBecause.js [hunk 14]
```diff
@@ -1,6 +1,6 @@
 module.exports = (errorMessage) => {
   // confirm we are in the right place
-  cy.url().should('include', '/appellant-submission/supporting-documents');
+  cy.url().should('include', '/appeal-householder-decision/supporting-documents');
```
Comment (a6software) [inline e2e-tests/cypress/support/appellant-submission-supporting-documents/confirmSupportingDocumentRejectedBecause.js:3]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: e2e-tests/cypress/support/appellant-submission-upload-application/confirmPlanningApplicationRejectedBecause.js [hunk 15]
```diff
@@ -1,6 +1,6 @@
 module.exports = (errorMessage) => {
   // confirm we are in the right place
-  cy.url().should('include', '/appellant-submission/upload-application');
+  cy.url().should('include', '/appeal-householder-decision/upload-application');
```
Comment (a6software) [inline e2e-tests/cypress/support/appellant-submission-upload-application/confirmPlanningApplicationRejectedBecause.js:3]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: e2e-tests/cypress/support/appellant-submission-your-details/confirmApplicantNameWasAccepted.js [hunk 16]
```diff
@@ -1,4 +1,4 @@
 module.exports = () => {
-  cy.url().should('include', '/appellant-submission/task-list');
+  cy.url().should('include', '/appeal-householder-decision/task-list');
```
Comment (a6software) [inline e2e-tests/cypress/support/appellant-submission-your-details/confirmApplicantNameWasAccepted.js:2]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: packages/forms-web-app/src/lib/views.js [hunk 17]
```diff
@@ -21,24 +21,24 @@ const VIEW = {
...
-    YOUR_DETAILS: 'appellant-submission/your-details',
-    WHO_ARE_YOU: 'appellant-submission/who-are-you',
-    APPLICANT_NAME: 'appellant-submission/applicant-name',
-    UPLOAD_APPLICATION: 'appellant-submission/upload-application',
-    UPLOAD_DECISION: 'appellant-submission/upload-decision',
-    CHECK_ANSWERS: 'appellant-submission/check-answers',
-    SUBMISSION: 'appellant-submission/submission',
-    CONFIRMATION: 'appellant-submission/confirmation',
-    SUBMISSION_INFORMATION: 'appellant-submission/submission-information',
+    TASK_LIST: 'appeal-householder-decision/task-list',
```
Comment (a6software) [inline packages/forms-web-app/src/lib/views.js:24]: please can line 23 be updated to `APPEAL_HOUSEHOLDER_DECISION` ? thanks

File: packages/forms-web-app/tests/unit/controllers/appellant-submission/appeal-statement.test.js [hunk 18]
```diff
@@ -71,7 +71,7 @@ describe('controllers/appellant-submission/appeal-statement', () => {
       });
     });
 
-    it('should redirect back to `/appellant-submission/appeal-statement` if validation passes but `i-confirm` not given', async () => {
+    it('should redirect back to `/appeal-householder-decision/appeal-statement` if validation passes but `i-confirm` not given', async () => {
```
Comment (a6software) [inline packages/forms-web-app/tests/unit/controllers/appellant-submission/appeal-statement.test.js:74]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: packages/forms-web-app/tests/unit/controllers/appellant-submission/application-number.test.js [hunk 19]
```diff
@@ -86,7 +86,7 @@ describe('controllers/appellant-submission/application-number', () => {
       });
     });
 
-    it('should redirect to `/appellant-submission/upload-application` if valid', async () => {
+    it('should redirect to `/appeal-householder-decision/upload-application` if valid', async () => {
```
Comment (a6software) [inline packages/forms-web-app/tests/unit/controllers/appellant-submission/application-number.test.js:89]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: packages/forms-web-app/tests/unit/controllers/appellant-submission/site-access.test.js [hunk 20]
```diff
@@ -90,7 +90,7 @@ describe('controllers/appellant-submission/site-access', () => {
       });
     });
 
-    it('should redirect to `/appellant-submission/site-access-safety` if `site-access` is `yes`', async () => {
+    it('should redirect to `/appeal-householder-decision/site-access-safety` if `site-access` is `yes`', async () => {
```
Comment (a6software) [inline packages/forms-web-app/tests/unit/controllers/appellant-submission/site-access.test.js:93]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: packages/forms-web-app/tests/unit/controllers/appellant-submission/site-location.test.js [hunk 21]
```diff
@@ -90,7 +90,7 @@ describe('controllers/appellant-submission/site-location', () => {
       });
     });
 
-    it('should redirect to `/appellant-submission/site-ownership` if valid', async () => {
+    it('should redirect to `/appeal-householder-decision/site-ownership` if valid', async () => {
```
Comment (a6software) [inline packages/forms-web-app/tests/unit/controllers/appellant-submission/site-location.test.js:93]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: packages/forms-web-app/tests/unit/controllers/appellant-submission/upload-application.test.js [hunk 22]
```diff
@@ -119,7 +119,7 @@ describe('controllers/appellant-submission/upload-application', () => {
       expect(res.redirect).toHaveBeenCalledWith(fakeNextUrl);
     });
 
-    it('should redirect to `/appellant-submission/supporting-documents` if valid', async () => {
+    it('should redirect to `/appeal-householder-decision/supporting-documents` if valid', async () => {
```
Comment (a6software) [inline packages/forms-web-app/tests/unit/controllers/appellant-submission/upload-application.test.js:122]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

File: packages/forms-web-app/tests/unit/controllers/appellant-submission/upload-decision.test.js [hunk 23]
```diff
@@ -138,7 +138,7 @@ describe('controllers/appellant-submission/upload-decision', () => {
       expect(res.redirect).toHaveBeenCalledWith(fakeNextUrl);
     });
 
-    it('should redirect to `/appellant-submission/task-list` if valid', async () => {
+    it('should redirect to `/appeal-householder-decision/task-list` if valid', async () => {
```
Comment (a6software) [inline packages/forms-web-app/tests/unit/controllers/appellant-submission/upload-decision.test.js:141]: please can the dir here be updated to reflect `appeal-householder-decision` - thanks

**Review by a6software — CHANGES_REQUESTED** [review]
Appreciate it's the same change over and over - sorry to cause a bunch of work.

**Review by swdpcomputing — APPROVED** [review]

**Comment by davehlewis** [comment]
Closed due to changes with how this ticket will work. For now only title changes

**Pull request closed without merging**